_opensearch/rest-api/snapshots/get-snapshot.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Get Snapshot
+parent: Snapshot APIs
+grand_parent: REST API reference
+nav_order: 6
+---
+
+## Get snapshot.
+
+Retrieves information about a snapshot.
+
+### Path parameters
+
+| Parameter | Data Type | Description |
+| :--- | :--- | :--- |
+| repository | String | The repository that contains the snapshot to retrieve. |
+| snapshot | Snapsnot | Snapshot to retrieve.
```

Comment:
SnapsHot

Reply:
Actually mistake on my part. It should be `String` instead of `Snapshot`. I made the change.